build.gradle

```diff
@@ -121,7 +121,7 @@ subprojects {
     }
 }
 
-project(':traverson4j-hc4') {
+project(':traverson4j-hc5') {
```

Comment:
Whats your thoughts on versioning this artifact, should we start this module from v1? Should we up the whole traverson4j project to v2, given this significant change?

Reply:
Yeah it feels like maybe we make a v2 release of the whole project which could also encompass the junit5 upgrade too

Reply:
also java 11 perhaps

Reply:
Ok cool, will look to building up a v2 release branch set